env/env.py

```diff
@@ -0,0 +1,255 @@
+import numpy as np
+import pandas as pd
+from gym.utils import seeding
+import gym
+from gym import spaces
+import matplotlib
+
+matplotlib.use("Agg")
+import matplotlib.pyplot as plt
+from stable_baselines3.common.vec_env import DummyVecEnv
+
+
+class CryptoPortfolioEnv(gym.Env):
+    """A single stock trading environment for OpenAI gym
+    Attributes
+    ----------
+        df: DataFrame
+            input data
+        stock_dim : int
+            number of unique stocks

```

Comment:
Esto va a ser 1 siempre o se pueden añadir mas? Lo digo por el comment `A single stock trading environment for OpenAI gym`